ng/src/web/components/icon/greenboneicon.js

```diff
@@ -24,14 +24,13 @@
 import React from 'react';
 
 import _ from 'gmp/locale.js';
-import {classes} from 'gmp/utils';
 
 import PropTypes from '../../utils/proptypes.js';
 
 import Icon from './icon.js';
 
 const GreenboneIcon = ({className, size = 'default', ...props}) => {
-  className = classes(className, 'greenbone-icon');
+  className = className + ' greenbone-icon';
```

Comment:
Just remove className completely. We shouldn't add class names anymore. Only for debugging purposes in glamorous components.

Reply:
It doesn't make sense to replace classses helper function by string concatenation. classes is still better and easier to use then using plain strings.